### PR TITLE
Add aws ec2 instructions

### PR DESCRIPTION
**Pull request by zmaalick**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,6 +65,49 @@ As of v2.0, the storage solution for making this data available alongside the no
...
+# Create an EC2 instance
+
+* Click on Launch instance
+* Choose Amazon Linux 2 AMI (HVM) 64-bit machine, click select
+* Choose t2.2xlarge and click next: configure instance details
+* Choose subnet default eu-west-2c
+* In IAM role choose existing trainings-ec2-dev role and click next: storage
+* 8 gb is fine, click next: add tags
+* Add following tags
+  * Name: Instance name
```
Comment (nhsavage) [inline README.md:80]: should this be set by the user to a unique name (and so in square brackets)

Comment (zmaalick) [reply README.md]: changed

File: README.md [hunk 2]
```diff
@@ -65,6 +65,49 @@ As of v2.0, the storage solution for making this data available alongside the no
...
+  * Tenable: FA
+  * ServiceOwner: [firstname.lastname]
+  * ServiceCode: PABCLT
+* add securitygroup, select an existing security group: IAStrainings-ec2-mo
+* Review and Launch and then select launch
```
[the remark this answers is not in the document]
Comment (zmaalick) [reply README.md:86]: It is intentional for a time being

File: README.md [hunk 3]
```diff
@@ -65,6 +65,49 @@ As of v2.0, the storage solution for making this data available alongside the no
...
+* It will prompt to set a key pair (to allow ssh). create a new key and download it. 
+
+It will create the instance. To see the running instance goto instances and instacne state will be "Running"
```
[the remark this answers is not in the document]
Comment (zmaalick) [reply README.md:89]: We will always use AdministratorAccess

File: README.md [hunk 4]
```diff
@@ -65,6 +65,49 @@ As of v2.0, the storage solution for making this data available alongside the no
 
 Data relating to the **CSSP 20CRDS** tutorials is held online in an Azure Blob Storage Service. To access this data user will need a valid shared access signature (SAS) token.  The data is in [Zarr](https://zarr.readthedocs.io/en/stable/) format and the total volume is ~2TB. The data is in hourly, 3 hourly, 6 hourly, daily and monthly frequencies stored seperatrely under the `metoffice-20cr-ds` container on MS-Azure. Monthly data only is also via [Zenodo](https://zenodo.org/record/2558135).
 
+
+## AWS 
+
+# Create an EC2 instance
```
[the remark this answers is not in the document]
Comment (zmaalick) [reply README.md:71]: added .. mine was already set, so didnt notice

File: README.md [hunk 5]
```diff
@@ -65,6 +65,49 @@ As of v2.0, the storage solution for making this data available alongside the no
...
+
+* Click on Launch instance
+* Choose Amazon Linux 2 AMI (HVM) 64-bit machine, click select
```
[the remark this answers is not in the document]
Comment (zmaalick) [reply README.md]: added

**Review by nhsavage — APPROVED** [review]
all seems good now

**Pull request merged**